examples/thermostat/thermostat-common/src/thermostat-delegate-impl.cpp

```diff
@@ -347,52 +339,61 @@ CHIP_ERROR ThermostatDelegate::RemoveFromThermostatSuggestionsList(uint8_t uniqu
     return CHIP_NO_ERROR;
 }
 
-uint8_t ThermostatDelegate::GetUniqueID()
+bool ThermostatDelegate::FindInList(uint8_t uniqueIDToFind)
```

Comment:
```suggestion
bool ThermostatDelegate::HaveSuggestionWithID(uint8_t uniqueIDToFind)
```
seems like a better name, no?